### PR TITLE
Hotfix: return UWP targets for back compat

### PR DESCRIPTION
**Pull request by abaranch**

### CODE REVIEW

File: src/Core/NuGet/Core/Package.nuspec [hunk 1]
```diff
@@ -67,5 +72,10 @@
     <file src="$configuration$\Core\Managed\Net46\Microsoft.ApplicationInsights.pdb" target="lib\net46" />
     <file src="$configuration$\Core\Managed\Net46\Microsoft.ApplicationInsights.xml" target="lib\net46" />
 
+    <!-- wp8: Binaries for Windows Phone 8.0 and 8.1 (Silverlight Runtime) projects -->
+    <file src="$configuration$\Core\Managed\DummyCore\Wp8Files\Microsoft.ApplicationInsights.dll" target="lib\wp8" />
+
+    <!-- portable-win81+wpa81: Windows Runtime projects (Phone & Store, Managed)-->
+    <file src="$configuration$\Core\Managed\DummyCore\Portable81Files\Microsoft.ApplicationInsights.dll" target="lib\portable-win81+wpa81" />
```
Comment (SergeyKanzhelev) [inline src/Core/NuGet/Core/Package.nuspec]: Do you need to define group `portable-win81+wpa81` above?

Comment (izikl) [inline src/Core/NuGet/Core/Package.nuspec]: Try this one as well:
<file src="$configuration$\Core\Managed\DummyCore\Portable81Files\Microsoft.ApplicationInsights.dll" target="lib\uap10.0" />

Comment (izikl) [inline src/Core/NuGet/Core/Package.nuspec]: lib\uap10.0

**Comment by dnduffy** [comment]
:)

**Comment by abaranch** [comment]
Closing this pull request without merging to master since we did not decide if we want to support this fix going forward. 2.0.1 was released from a hotfix branch.

**Pull request closed without merging**